nautobot/core/ui/object_detail.py

```diff
@@ -2130,8 +2143,8 @@ def __init__(
                     max_display_count=100,  # since there isn't a separate list view for ContactAssociations!
                     # TODO: we should provide a standard reusable component template for bulk-actions in the footer
                     footer_content_template_path="components/panel/footer_contacts_table.html",
-                    header_extra_content_template_path=None,
-                    label="Contacts/Teams",
+                    enable_related_link=False,
+                    table_title="Contacts/Teams",
```

Comment:
Huh. What's the difference between `label` and `table_title` in practice? Do we have two ways of doing the same thing?

Reply:
I'm not sure the reason behind having both, but `label` is [rendered on the panel here](https://github.com/nautobot/nautobot/blob/f61e9b426f40ff5efd3a1c963ea968f1a4f73e87/nautobot/core/templates/components/panel/panel.html#L4-L6), and the `table_title` is rendered inside the `header_extra_content` [here](https://github.com/nautobot/nautobot/blob/f61e9b426f40ff5efd3a1c963ea968f1a4f73e87/nautobot/core/templates/components/panel/header_extra_content_table.html#L3).